### PR TITLE
use bookingsync-stylecheck

### PR DESCRIPTION
**Pull request by Mandaryn**

### CODE REVIEW

File: spec/bookingsync/api/client/bathrooms_spec.rb [hunk 1]
```diff
@@ -19,7 +18,7 @@
   end
 
   describe ".create_bathroom", :vcr do
-    let(:attributes) {{
+    let(:attributes) { {
       name_en: "New bathroom",
       wc: true
     }}
```
Comment (ZenCocoon) [inline spec/bookingsync/api/client/bathrooms_spec.rb]: Funny that is did not catch this one, any reason?

File: spec/bookingsync/api/client/bedrooms_spec.rb [hunk 2]
```diff
@@ -19,7 +18,7 @@
   end
 
   describe ".create_bedroom", :vcr do
-    let(:attributes) {{
+    let(:attributes) { {
       name_en: "New bedroom",
       sofa_beds_count: 2
     }}
```
Comment (ZenCocoon) [inline spec/bookingsync/api/client/bedrooms_spec.rb]: Not catch either

File: spec/bookingsync/api/client/bathrooms_spec.rb (hunk 1 above)
Comment (Mandaryn) [inline spec/bookingsync/api/client/bathrooms_spec.rb]: noone said its perfect, but good at doing 99% of the job

**Comment by Mandaryn** [comment]
fixed to use do end blocks and in some cases made a single line hash

**Pull request merged**